lib/App/Yath/Command/runner.pm

```diff
@@ -277,7 +277,10 @@ sub build_init_state {
 
     # Make sure our specified includes are at the front of the list.
     my %seen;
-    @INC = grep { !$seen{$_}++ } $job->unsafe_inc ? ('.') : (), map { clean_path( $_ ) } $job->includes, @INC;
+    @INC = grep { !$seen{$_}++ } $job->unsafe_inc ? ('.') : (), map {
+        my $p = clean_path( $_ );
+        index($p, '/', 0 ) != 0 || -d $p ? $p : ()
```

Comment:
this can probably be converted to a simple `-d $p ? $p : ()` as clean_path is going to guarantee an absolute PATH